instrumentation/opentelemetry-instrumentation-starlette/src/opentelemetry/instrumentation/starlette/package.py

```diff
@@ -13,6 +13,6 @@
 # limitations under the License.
 
 
-_instruments = ("starlette ~= 0.13.0",)
+_instruments = ("starlette >= 0.13, <0.15",)
```

Comment:
0.15 is too old, FYI

Reply:
We know it's old but it's what the instrumentation is currently able to instrument :sweat_smile:

Reply:
That's the same as saying that Starlette doesn't have an instrumentation package. That version is 4 years old.

Reply:
But I was just pointing out.

Reply:
Ya it probably means many people are using a buggy instrumentation. Let's take this discussion into https://github.com/open-telemetry/opentelemetry-python-contrib/issues/3317